airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -538,6 +531,30 @@ def test_additional_properties_is_true(self, actual_connector_spec: ConnectorSpe
                 [additional_properties_value is True for additional_properties_value in additional_properties_values]
             ), "When set, additionalProperties field value must be true for backward compatibility."
 
+    # This test should not be part of TestSpec because it's testing the connector's docker image content, not the spec itself
+    # But it's cumbersome to declare a separate, non configurable, test class
+    # See https://github.com/airbytehq/airbyte/issues/15551
+    def test_image_labels(self, docker_runner: ConnectorRunner, connector_metadata: dict):
+        """Check that connector's docker image has required labels"""
+        assert docker_runner.labels.get("io.airbyte.name"), "io.airbyte.name must be set in dockerfile"
+        assert docker_runner.labels.get("io.airbyte.version"), "io.airbyte.version must be set in dockerfile"
+        assert (
+            docker_runner.labels.get("io.airbyte.name") == connector_metadata["data"]["dockerRepository"]
+        ), "io.airbyte.name must be equal to dockerRepository in metadata.yaml"
```

Comment:
One day, it would be nice to not need to modify the docker labels at all.  I'm curious if we even still need them.  I don't think the platform uses them at all...

Reply:
Agreed! It's just a good enough way for me right now to try to detect and understand the CI problem we might have with caching.
The metadata.yaml is not packaged into the connector image on the Python connectors, if we revamped as planned the build logic for python connector I think we shall package metadata.yaml in the connector image.
Wdyt about a new connector command like `metadata` that would output the connector metadata?

Reply:
I like it.  Then I can `airbybye-ci connectors metadata --name source-foo | jq` :D

Reply:
@evantahler I meant a connector command:
`docker run airbyte/pokeapi metadata | jq`